parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java

```diff
@@ -471,6 +484,12 @@ public Builder withBloomFilterNDV(String columnPath, long ndv) {
       return this;
     }
 
+    public Builder withBloomFilterFPP(String columnPath, double fpp) {
```

Comment:
what happens if this value is set, but the BF is not enabled? (general / per-column)

Reply:
This value will be silently ignored.